Modules/System/.AL-Go/settings.json

```diff
@@ -0,0 +1,16 @@
+{
+    "country":  "base",
+    "appFolders":  [
+        "..\\System"
+    ],
+    "testFolders":  [
+        "..\\System Test Libraries",
+        "..\\System Tests"
+    ],
+    "buildModes": [
+        "Translated"
+    ],
+    "enablePerTenantExtensionCop":  false,
+    "enableCodeCop": true, 
+    "doNotRunTests": true
```

Comment:
Could _country, enablePerTenantExtensionCop, enableCodeCop_ and _doNotRunTests_ be moved to the repo settings? If we have the same settings across all AL-GO projects, we might as well move them up in the settings hierarchy

Reply:
Actually... Maybe not doNotRunTests. I guess we don't want to run tests for this AL-GO project, but for the _Modules_ project we do.

Reply:
Moved all but `doNotRunTests` to the common AL-Go settings.